README.md

```diff
@@ -48,82 +48,77 @@ ARD Examples
-| ARM    | Placebo              | AGEGR1   | \<65           | n         | n          | 14                | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | \<65           | p         | %          | 0.162790697674419 | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | \>80           | n         | n          | 30                | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | \>80           | p         | %          | 0.348837209302326 | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | 65-80          | n         | n          | 42                | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | 65-80          | p         | %          | 0.488372093023256 | NA      | NA    |
-| ARM    | Placebo              | AGEGR1   | NA             | N         | N          | 86                | NA      | NA    |
-| ARM    | Xanomeline High Dose | AGEGR1   | \<65           | n         | n          | 11                | NA      | NA    |
-| ARM    | Xanomeline High Dose | AGEGR1   | \<65           | p         | %          | 0.130952380952381 | NA      | NA    |
-| ARM    | Xanomeline High Dose | AGEGR1   | \>80           | n         | n          | 18                | NA      | NA    |
-
-``` r
+ard_continuous(ADSL, by = "ARM", variables = c("AGE", "BMIBL"))
+#> {cards} data frame: 48 x 10
+#>    group1 group1_level variable stat_name stat_label statistic
+#> 1     ARM      Placebo      AGE         N          N        86
+#> 2     ARM      Placebo      AGE      mean       Mean    75.209
+#> 3     ARM      Placebo      AGE        sd         SD      8.59
+#> 4     ARM      Placebo      AGE    median     Median        76
+#> 5     ARM      Placebo      AGE       p25  25th Per…        69
+#> 6     ARM      Placebo      AGE       p75  75th Per…        82
+#> 7     ARM      Placebo      AGE       min        Min        52
+#> 8     ARM      Placebo      AGE       max        Max        89
+#> 9     ARM      Placebo    BMIBL         N          N        86
+#> 10    ARM      Placebo    BMIBL      mean       Mean    23.636
+#> ℹ 38 more rows
+#> ℹ Use `print(n = ...)` to see more rows
+#> ℹ 4 more variables: context, statistic_fmt_fn, warning, error
+
+ard_categorical(ADSL, by = "ARM", variables = c("AGEGR1", "SEX"))
+#> {cards} data frame: 36 x 11
+#>    group1 group1_level variable variable_level stat_name stat_label statistic
+#> 1     ARM      Placebo   AGEGR1            <65         n          n        14
+#> 2     ARM      Placebo   AGEGR1            <65         p          %     0.163
+#> 3     ARM      Placebo   AGEGR1            >80         n          n        30
+#> 4     ARM      Placebo   AGEGR1            >80         p          %     0.349
+#> 5     ARM      Placebo   AGEGR1          65-80         n          n        42
+#> 6     ARM      Placebo   AGEGR1          65-80         p          %     0.488
+#> 7     ARM      Placebo   AGEGR1                        N          N        86
```

Comment:
I think the readme needs re-rendering after the last update :)

Reply:
ahhh good catch! thank you!